app/src/main/java/org/coepi/android/ui/formatters/DateFormatters.kt

```diff
@@ -6,10 +6,27 @@ import java.util.Date
 
 object DateFormatters {
     val dotFormatter = DotDateFormatter()
+    val timeFormatter = TimeFormatter()
+    val monthDayFormatter = MonthDayFormatter()
 }
 
 class DotDateFormatter {
     @SuppressLint("SimpleDateFormat")
-    val format = SimpleDateFormat("dd.MM.yyyy")
-    fun format(date: Date): String = format.format(date)
+    val formatDayMonthYear = SimpleDateFormat("dd.MM.yyyy")
+
+    fun formatDayMonthYear(date: Date): String = formatDayMonthYear.format(date)
+}
+
+class TimeFormatter {
```

Comment:
I'd chose a more specific name, as we may have other time formatters. `HoursMinsFormatter` or similar.